app/api-v1/controllers/Order/index.js

```diff
@@ -12,8 +18,37 @@ module.exports = {
     get: async () => {
       return { status: 500, response: { message: 'Not Implemented' } }
     },
-    create: async () => {
-      return { status: 500, response: { message: 'Not Implemented' } }
+    create: async (req) => {
+      const { id } = req.params
+      if (!id) throw new BadRequestError('missing params')
+
+      const [order] = await db.getOrder(id)
+      if (!order) throw new NotFoundError('order')
+
+      const selfAddress = await idenity.getMemberBySelf()
+      if (!selfAddress) throw new IdentityError()
+
+      const transaction = await db.insertOrderTransaction(id)
+      const payload = {
+        inputs: [],
+        outputs: [
+          {
+            roles: {
+              Owner: selfAddress,
```

Comment:
Needs to be `Owner: order.supplier` as they're the next party that will transact with the `order` token (this will change when we remove the [restriction](https://github.com/digicatapult/dscp-node/blob/1deadf15a7ae6660ac6f1aedcfb0cc64f02fc39d/pallets/simple-nft/src/lib.rs#L201) that only `Owner` (the default role) can burn a token)

Reply:
thanks!

Reply:
made a ticket for removing the restriction from the node https://digicatapult.atlassian.net/browse/IN-310